spec/imsc-hrm.html

```diff
@@ -657,13 +657,14 @@ <h2>Paint Text</h2>
           </tr>
 
           <tr>
-            <td style="text-align: center;"><em>Block property (see [[!UNICODE]]) for the character of g<sub>i</sub></em></td>
+            <td style="text-align: center;"><em>Script property (see Standard Annex #24 at [[!UNICODE]]) for the character of
```

Comment:
I think you can just use `[[!UAX24]]` to refer to `Unicode® Standard Annex #24` in ReSpec.